esp/esp/cache/views.py

```diff
@@ -36,9 +36,17 @@
 from esp.cache.registry import all_caches
 from esp.users.models import admin_required
 from esp.web.util.main import render_to_response
+from django.shortcuts import redirect
+from django.core.urlresolvers import reverse
 
 @admin_required
 def view_all(request):
-    caches = sorted(all_caches.values(), key=lambda c: c.name)
+    caches = sorted(all_caches, key=lambda c: c.name)
     cache_data = [{'pretty_name': cache.pretty_name, 'hit_count': cache.hit_count, 'miss_count': cache.miss_count} for cache in caches]
```

Comment:
I just tried making this change, and it broke something. I think maybe the template gets confused because it thinks the cache objects are callable?